src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,141 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://pants.readme.io/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0.dev6 (7/28/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Eliminate most rule registration boilerplate. (#10477)
+  `PR #10477 <https://github.com/pantsbuild/pants/pull/10477>`_
+
+* Remove unused `util/` and `base/` code (#10462)
+  `PR #10462 <https://github.com/pantsbuild/pants/pull/10462>`_
+
+* Rename `UrlToFetch` to `DownloadFile` (#10468)
+  `PR #10468 <https://github.com/pantsbuild/pants/pull/10468>`_
+
+* Change the `PathGlobs`, `SnapshotSubset`, and `UrlToFetch` intrinsics to return `Digest` (#10449)
+  `PR #10449 <https://github.com/pantsbuild/pants/pull/10449>`_
+
+* Unify accessing options for a `GoalSubsystem` with `Subsystem` (#10456)
+  `PR #10456 <https://github.com/pantsbuild/pants/pull/10456>`_
+
+* Rename `Workspace.materialize_directory()` to `Workspace.write_digest()` (#10448)
+  `PR #10448 <https://github.com/pantsbuild/pants/pull/10448>`_
+
+* Remove unused `SingleFileExecutable` (#10443)
+  `PR #10443 <https://github.com/pantsbuild/pants/pull/10443>`_
```

Comment:
These all seem like refactoring, given that the rules API is nowhere near stable. The motivation for moving there is so that the release notes are smaller for non-plugin authors upgrading from 1.30 to 2.0.